deploy/docker-compose/docker-compose-fixed-instrumented.yml

```diff
@@ -75,7 +75,7 @@ services:
       - DD_TRACE_ANALYTICS_ENABLED=true
       - DD_PROFILING_ENABLED=true
       - DD_VERSION=1.0
-    image: "ddtraining/advertisements-service:latest"
+    image: "ddtraining/advertisements-fixed:latest"
```

Comment:
Did we set up a GCP container registry yet? Do we want to link to that instead?

Reply:
Not right now. I'd rather get v1.0 published to Docker before we start messing with cross-platform publishing just yet.